Dockerfile

```diff
@@ -5,29 +5,44 @@ RUN apt-get update
 RUN apt-get --no-install-recommends install --yes \
     libaio1 libaio-dev xmlsec1 libffi-dev \
     libldap2-dev libsasl2-dev \
-    build-essential libmysqlclient-dev git
+    build-essential default-libmysqlclient-dev git
 
 WORKDIR /tmp/
 
-COPY student_explorer/dependencies/oracle-instantclient12.1-*.deb /tmp/
-RUN dpkg -i oracle-instantclient12.1-*.deb
-ENV ORACLE_HOME /usr/lib/oracle/12.1/client64
-ENV LD_LIBRARY_PATH /usr/lib/oracle/12.1/client64/lib
-RUN rm oracle-instantclient12.1-*.deb
+COPY requirements.txt /tmp/
 
-RUN pip install cx_Oracle==5.3
-RUN pip install gunicorn
+RUN pip install -r requirements.txt && \
+    pip install gunicorn
 
-COPY requirements.txt /tmp/
-RUN pip install -r requirements.txt
+#https://github.com/jwilder/dockerize
+ENV DOCKERIZE_VERSION v0.6.1
+RUN curl -sLO "https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz" \
```

Comment:
This is "dockerize" a script that this individual wrote. It is at https://github.com/jwilder/dockerize This is their recommended install instructions.

The main use for this script is to ensure that the database container is running before starting up the application. This is what I've also used for student dashboard. It's really just used for the local instance since MiDatabase (should) always be running and available.

Reply:
I wonder whether we shall maintain a local copy of this dockerize script in our repository, just to avoid the situation that the script would be removed from external sources.

Maybe can track as another issue.

Reply:
Yes, I agree we should fork this repo into out `github t&L` repo and use it that way we can up pull it from ours and ensure a local copy

Reply:
I created an issue to switch dockerize to wait-port https://github.com/tl-its-umich-edu/student-dashboard-django/issues/277

This looks like a better binary, and it's released to npm.